stl/src/ppltasks.cpp

```diff
@@ -235,7 +234,7 @@ namespace Concurrency {
             } else {
                 ComCallData callData;
                 ZeroMemory(&callData, sizeof(callData));
-                callData.pUserDefined = &_Func;
+                callData.pUserDefined = reinterpret_cast<void*>(&_Func);

```

Comment:
what's going on here? This cast (from std::function<void __cdecl()>* to void*) is automatic and reinterpret_cast substantially increases it's power. afaict this is not a cast from a function pointer type to void* (which is not implicit, but that's not what's happening here)

Reply:
this was inherited and later changed by Stephan (Oct 19, 2021) - will remove

Reply:
resynced with internal PR